build.yaml

```diff
@@ -1,4 +1,9 @@
+# To understand how to use the build/config system, read the docs in docs/build.md.
+
 webhookVersion: 104.0.0+up0.5.0-rc6
 cspAdapterMinVersion: 103.0.0+up3.0.0
 defaultShellVersion: rancher/shell:v0.1.22
+systemChartDefaultBranch: dev-v2.9
+chartDefaultBranch: dev-v2.9
+kdmBranch: dev-v2.9
```

Comment:
Question: is it true that we will need to update those values at two places: this file and the file `pkg/buildconfig/constants.go` ?

Reply:
The constants file is for use of these values in any Go code (if and when needed). But you need to change only the `build.yaml` file. Then you run `go generate`, and the tool updates the constants file for you. But changes to both files need to be committed.